src/components/orders/crud/OrderDetail.js

```diff
@@ -20,10 +20,22 @@ export default function OrderDetail(props) {
 		fetchData()
 	}, [])
 
+	const getProductsListElements = () => {
+		return productsList.map(product => {
+			return (
+				<Row>
+					<DetailElement header="Product Name:" value={product.name} />
```

Comment:
you can use Col inside this Row if you want

Reply:
let's leave it like this until building beautiful design